sycl/doc/extensions/proposed/sycl_ext_oneapi_usm_malloc_properties.asciidoc

```diff
@@ -0,0 +1,1468 @@
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
```

Comment:
Did you intend to add a table of contents?  I don't think I've seen it on any of the other extensions.

Reply:
I removed these lines. I don't want a table of contents.